src/components/enter-password/tests/enter-password-controller.test.ts

```diff
@@ -115,6 +115,53 @@ describe("enter password controller", () => {
   });
 
   describe("enterPasswordPost", () => {
+    it("can send the journeyType when sending the password", async () => {
+      const fakeService: EnterPasswordServiceInterface = {
+        loginUser: sinon.fake.returns({
+          data: {
+            redactedPhoneNumber: "3456",
+            mfaRequired: true,
+            consentRequired: false,
+            latestTermsAndConditionsAccepted: true,
+            mfaMethodVerified: true,
+            mfaMethodType: "SMS",
+          },
+          success: true,
+        }),
+      } as unknown as EnterPasswordServiceInterface;
+
+      const fakeMfaService: MfaServiceInterface = {
+        sendMfaCode: sinon.fake.returns({
+          success: true,
+        }),
+      } as unknown as MfaServiceInterface;
+
+      res.locals.sessionId = "123456-djjad";
+      res.locals.clientSessionId = "00000-djjad";
+      res.locals.persistentSessionId = "dips-123456-abc";
+      req.session.user = {
+        email: "joe.bloggs@test.com",
+        reauthenticate: "test_data",
+      };
+      req.body["password"] = "password";
+
+      await enterPasswordPost(
+        false,
+        fakeService,
+        fakeMfaService
+      )(req as Request, res as Response);
+
+      expect(fakeService.loginUser).to.have.calledWith(
```

Comment:
Probably the same resolutions are required here as described in https://github.com/govuk-one-login/authentication-frontend/pull/1290#discussion_r1455820477.